CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - On macOS, there's a ToggleSimpleFullscreen action which allows switching to
     fullscreen without occupying another space
 - A new window option `startup_mode` which controls how the window is created
+- Alacritty now loads its icon on unix platforms, except macOS
```

Comment:
This statement is a bit misleading, since this change should be completely unrelated to Wayland for example.

I'd be a bit more specific here and say that the `_NET_WM_ICON` property is set on X11 now, allowing for WMs to show icons in titlebars.